docs/pipelines/access-control/index.md

```diff
@@ -0,0 +1,29 @@
+# Access control
+
+Gruntwork Pipelines grants permissions by defining three [GitHub Teams](https://docs.github.com/en/organizations/organizing-members-into-teams/about-teams), which should map to three separate personas in your organization. Each team and its permissions are designed to grant the _least possible permissions_ to each individual (or machine user) in your organization for them to be able to perform changes to your infrastructure.
```

Comment:
Should we explicitly mention and link to the [principle of least privilege](https://en.wikipedia.org/wiki/Principle_of_least_privilege)?

Reply:
Yeah

Reply:
Added in [b52d6e5](https://github.com/gruntwork-io/docs/pull/1099/commits/b52d6e525b46856cd2731e09c89f401baf7170fe)